Revise the positionConfig contents (#650)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/positionConfig/at/at.md

---
---
##### shortDescription

> Specifies a side or corner of the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) at which the overlay element should be positioned. 

Reviewer: ```suggestion
Specifies the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of)'s side or corner where the overlay element should be positioned.
```

---

> To set this option, use an object with the **x** and **y** fields or one of the string shortcuts listed in the accepted values.

Reviewer: Can you be more specific than "use an object with the..."?
For example, do you mean:
To set this option, specify the object's **x** and **y** fields, or use a string shortcut from the accepted values list.

If not, please suggest a revised version.

Reviewer: Did you see my comment above?
